cloudify_agent/shell/commands/daemons.py

```diff
@@ -386,3 +424,51 @@ def _parse_custom_options(options):
         parsed[key] = value
 
     return parsed
+
+
+def _parse_security_settings(attributes):
+    updated_values = {}
+
+    attributes['security_enabled'] = api_utils.get_bool_or_default(
+        attributes.get('security_enabled'), defaults.SECURITY_ENABLED)
+
+    attributes['verify_rest_certificate'] = api_utils.get_bool_or_default(
```

Comment:
not required at this point anymore, has been used earlier to deduce other values (e.g. port) that are set at this stage